submissions/chernetskyi8704/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,189 @@
+            </label>
+
+            <div class="popup">
+              <ul class="popup__list">
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img
+                      class="popup__img"
+                      src="img/calendar.png"
+                      alt="Calendar"
+                    />
+                    <p class="link__text">Calendar</p></a
+                  >
+                </li>
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img class="popup__img" src="img/mail.png" alt="" />
+                    <p class="link__text">Mail</p></a
+                  >
+                </li>
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img class="popup__img" src="img/news.png" alt="" />
+                    <p class="link__text">News</p></a
+                  >
+                </li>
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img class="popup__img" src="img/maps.png" alt="" />
+                    <p class="link__text">Map</p></a
+                  >
+                </li>
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img class="popup__img" src="img/files.png" alt="" />
+                    <p class="link__text">Files</p></a
+                  >
+                </li>
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img class="popup__img" src="img/contacts.png" alt="" />
+                    <p class="link__text">Contacts</p></a
+                  >
+                </li>
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img class="popup__img" src="img/classroom.png" alt="" />
+                    <p class="link__text">Classroom</p></a
+                  >
+                </li>
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img class="popup__img" src="img/tanslation.png" alt="" />
+                    <p class="link__text">Translation</p></a
+                  >
+                </li>
+                <li class="popup__item">
+                  <a class="popup__link link" href=""
+                    ><img class="popup__img" src="img/documents.png" alt="" />
+                    <p class="link__text">Documents</p></a
+                  >
+                </li>
+              </ul>
+              <input
+                type="checkbox"
+                class="popup__more-checkbox"
+                id="popup__more-checkbox"
+              />
+              <label for="popup__more-checkbox" class="popup__more-label link"
+                >More</label
+              >
```

Comment:
When the second part of the popup is an open button `More` must be hidden.

Reply:
Thanks for the comment. I changed the display of the More button to none when It clicked.